Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```diff
@@ -0,0 +1,1096 @@
+      secret: false
+    - default: false
+      description: Display name
+      isArray: false
+      name: displayname
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether Duo two-factor authentication is enabled
+      isArray: false
+      name: duotwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: E-mail
+      isArray: false
+      name: emailaddress
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether the user account is enabled
+      isArray: false
+      name: enabled
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether FIDO2 two-factor authentication is enabled
+      isArray: false
+      name: fido2twofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: GroupOwners(integer)
+      isArray: false
+      name: groupowners
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: IsApplicationAccount
+      isArray: false
+      name: isapplicationaccount
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: isGroupOwnerUpdate
+      isArray: false
+      name: isgroupownerupdate
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether the user is locked out
+      isArray: false
+      name: isLockedout
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Number of login failures
+      isArray: false
+      name: loginfailures
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether OATH two-factor authentication is enabled
+      isArray: false
+      name: oathtwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Password
+      isArray: false
+      name: password
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether RADIUS two-factor authentication is enabled
+      isArray: false
+      name: radiustwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: RADIUS username
+      isArray: false
+      name: radiususername
+      required: false
+      secret: false
+    - default: false
+      description: timeOptionId (integer)
```

Comment:
same here

Reply:
@guykeller 

documentation API
[API](https://updates.thycotic.net/secretserver/restapiguide/10.8/index.html)

Reply:
I'm not sure why the API link is relevant, can you explain?
No matter what the API says, the description should at least include a short plain English description of the argument, that will explain what it means and how it should be used

Reply:
I will request a more detailed description of the API from Thycotik, if they have one.